packages/nuxeo-coldstorage-package/src/main/resources/install.templates.coldstorage/nuxeo.defaults

```diff
@@ -0,0 +1,3 @@
+## DO NOT EDIT THIS FILE, USE nuxeo.conf  ##
```

Comment:
What is this file for?

Reply:
I was thinking that file should be added even empty

Reply:
If there is no template needed, the file is not needed.
BTW, even if the file should be here, it should not be in `install.templates.coldstorage/` but in `install/templates/coldstorage/` to actually make it work
=> it's really not needed :p